spec/controllers/tour_requests_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+
+RSpec.describe TourRequestsController, type: :controller do
+  let(:user) {
```

Comment:
let khác gì let!

Reply:
let có lazy-evaluated, tức là nó sẽ ko chạy cho đến khi lần đầu tiên được gọi đến, còn let! sẽ chạy trước mỗi it